.github/workflows/ci.yml

```diff
@@ -24,6 +24,11 @@ jobs:
           ghc-version: ${{ matrix.ghc }}
           cabal-version: 3.6.2.0
       - run: cabal update
+      # TODO: remove once hedgehog is compatible with ghc-9.6
+      - name: "Support ghc-9.6"
+        if: matrix.ghc == '9.6.1'
+        run: |
+          echo "allow-newer: hedgehog:template-haskell" >> cabal.project.local
```

Comment:
With the new support for conditionals in `cabal.project` files you can now do this much more transparently:

```
if impl(ghc >= 9.6)
  allow-newer: hedgehog:template-haskell
```

Reply:
Nice, done.